pandas_datareader/binance/__init__.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+from pandas_datareader.base import _BaseReader
+import requests
+
+BINANCE_BASE_URL = "https://api.binance.com"
+
+
+class BinanceReader(_BaseReader):
+    """Get data for the given name from Binance."""
+
+    _format = None
+
+    _intervals = {
+        "ONEMINUTE": "1m",
+        "THREEMINUTE": "3m",
+        "FIVEMINUTE": "5m",
+        "FIFTEENMINUTE": "15m",
+        "THIRTYMINUTE": "30m",
+        "ONEHOUR": "1h",
+        "TWOHOUR": "2h",
+        "FOURHOUR": "4h",
+        "SIXHOUR": "6h",
+        "EIGHTHOUR": "8h",
+        "TWELVEHOUR": "12h",
+        "ONEDAY": "1d",
+        "THREEDAY": "3d",
+        "ONEWEEK": "1w",
+        "ONEMONTH": "1M",
+    }
+
+    def __init__(
+        self,
+        symbols=None,
+        start=None,
+        end=None,
+        retry_count=3,
+        pause=0.1,
+        session=None,
+        interval="ONEDAY",
+        limit=500,
+    ):
+        super(BinanceReader, self).__init__(
+            symbols=symbols,
+            start=start,
+            end=end,
+            retry_count=retry_count,
+            pause=pause,
+            session=session,
+        )
+        self._interval = interval
+        self._limit = limit
+
+    @property
+    def url(self):
+        """API URL"""
+        return BINANCE_BASE_URL + "/api/v1/klines"
+
+    def clean_data(self, data):
+        dataFrame = pd.DataFrame(
+            data.json(),
+            columns=[
+                "Open time",
+                "Open",
+                "High",
+                "Low",
+                "Close",
+                "Volume",
+                "Close time",
+                "Quote asset volume",
+                "Number of trades",
+                "Taker buy base asset volume",
+                "Taker buy quote asset volume",
+                "ignore",
+            ],
+        )
+        dataFrame = dataFrame.drop(columns="ignore")
+        return dataFrame
+
+    def get_interval(self):
+        return self._interval
+
+    def convert_time_to_miliseconds(self, dt):
```

Comment:
This is not python 2 compatible. Use ``time`` instead:

```python
int(time.mktime(dt.timetuple()) * 1000)
```

Reply:
Also same comment re: leading underscore

Reply:
This has been updated and checked.